LayoutTests/inspector/runtime/callFunctionOn-awaitPromise.html

```diff
@@ -0,0 +1,263 @@
+                returnByValue: true,
+                objectId: thisObject.result.objectId
+            });
+            InspectorTest.expectTrue(result.wasThrown, "Should throw an error.");
+            InspectorTest.expectEqual(result.result.value, "My error", `Should return the error.`);
+        },
+    });
+
+    suite.addTestCase({
+        name: "NoAwaitAsyncFunction",
+        description: "Check that callFunctionOn with awaitPromise=false works with an asynchronous function.",
+        async test() {
+            let thisObject = await RuntimeAgent.evaluate.invoke({expression: `myObject`});
+            let result = await RuntimeAgent.callFunctionOn.invoke({
+                functionDeclaration: `async function() { await new Promise((fulfill, reject) => setTimeout(fulfill, 10)); return this; }`,
+                awaitPromise: false,
+                returnByValue: true,
+                objectId: thisObject.result.objectId
+            });
+            InspectorTest.expectShallowEqual(result.result.value, {}, `Should return empty object.`);
+        },
+    });
+
+    suite.addTestCase({
+        name: "NoAwaitPromiseRejection",
+        description: "Check that callFunctionOn with awaitPromise=false works with an asynchronous function.",
+        async test() {
+            let thisObject = await RuntimeAgent.evaluate.invoke({expression: `myObject`});
+            let result = await RuntimeAgent.callFunctionOn.invoke({
+                functionDeclaration: `function() { return new Promise((fulfill, reject) => setTimeout(reject, 10, this)); }`,
+                awaitPromise: false,
+                returnByValue: true,
+                objectId: thisObject.result.objectId
+            });
+            InspectorTest.expectFalse(result.wasThrown, "Should not throw an error.");
+            InspectorTest.expectShallowEqual(result.result.value, {}, `Should return empty object.`);
+        },
+    });
+
+    suite.addTestCase({
+        name: "NoAwaitWrongObjectIdError",
+        description: "Check that callFunctionOn returns error string when passed unknown object id.",
+        async test() {
+            let thisObject = await RuntimeAgent.evaluate.invoke({expression: `myObject`});
+            let parsedId = JSON.parse(thisObject.result.objectId);
+            parsedId.id = undefined;
+            let objectId = JSON.stringify(parsedId);
+            try {
+                await RuntimeAgent.callFunctionOn.invoke({
+                    functionDeclaration: `async function() { await new Promise((fulfill, reject) => setTimeout(fulfill, 10)); return this; }`,
+                    awaitPromise: false,
+                    returnByValue: true,
+                    objectId
+                });
+                InspectorTest.fail("Protocol error was not thrown.");
+            } catch (e) {
+                InspectorTest.expectTrue(e.message.includes('Could not find object with given id'), "Received protocol error");
```

Comment:
Style: we prefer `"` instead of `'`

ditto below

Reply:
Dang, switching from one style to the another is not so easy. Updated all occurrences.